contracts/utils/contracts/Bitfield.sol

```diff
@@ -44,7 +45,6 @@ library Bitfield {
 
         bitfield = new uint256[](prior.length);
         uint256 found = 0;
-        uint256 length = prior.length * 256;
 
         for (uint256 i = 0; found < n; i++) {
             bytes32 randomness = keccak256(abi.encode(seed + i));
```

Comment:
I think this random algorithm is not good.
Is there any way that we can make sure to select N random bits with a limited steps?

Reply:
Such as we can use a prime number to make a permutation of the sorted number. We only need N steps to select a random sequence.

Reply:
https://github.com/darwinia-network/darwinia-bridge-sol/commit/ac2ed7161aa8c87732890da07a72aa7a716a0539